terraform/environments/hmpps-oem/locals.tf

```diff
@@ -17,7 +17,6 @@ locals {
     enable_image_builder                         = true
     enable_ec2_cloud_watch_agent                 = true
     enable_ec2_self_provision                    = true
-    enable_ec2_oracle_enterprise_manager         = true
```

Comment:
I think if we set enable_ec2_oracle_enterprise_managed_server = true instead it will simplify a lot of stuff.